src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/profileSettings/profileProperties/profilePropertyEditor/index.jsx

```diff
@@ -52,6 +54,10 @@ class ProfilePropertyEditor extends Component {
         props.dispatch(SiteBehaviorActions.getProfileProperty(props.propertyId, props.portalId));
     }
 
+    componentDidMount() {
+        ReactDOM.findDOMNode(this).querySelectorAll("input")[0].focus();
```

Comment:
Isn't `querySelector` a better choice if you're only interested in one element?

Also, please add a null check before `focus`. Just in case